components/suggest/src/suggest.udl

```diff
@@ -72,7 +72,8 @@ interface Suggestion {
     );
     Yelp(
         string url,
-        string title
+        string title,
+        boolean subject_exact_match
```

Comment:
Thought / question: For Pocket, we have a `boolean is_top_pick` field, and decide in the component what that means (e.g., if the keyword is a high-confidence keyword). Do you think it makes sense to do the same for Yelp?

You mentioned that we use `subject_exact_match` to decide whether to use the Top Pick treatment or not, so I'm wondering, does it make sense to call this `is_top_pick`, and set it to `true` / `false` in `yelp.rs` based on any criteria we might have now or in the future.

Reply:
Yeah, as you said, this means the same meaning for now. The reason I named this was that I thought it would be better to return only the facts as much as possible here and let the client interpret them. But, for consistency, better to use `is_top_pick`, as you said. Okay, I will rename it with `is_top_pick`. Thanks!